src/PackageVersions/Installer.php

```diff
@@ -137,8 +137,10 @@ private static function writeVersionClassToFile(string $versionClassSource, Comp
 
         $io->write('<info>ocramius/package-versions:</info>  Generating version class...');
 
-        file_put_contents($installPath, $versionClassSource);
-        chmod($installPath, 0664);
+        $installPathTmp = $installPath . '_' . microtime();
```

Comment:
Instead of `microtime()`, this is where I'd use `uniqid(..., true)`, which is designed for these scenarios